Model/System/Config/Source/Product/Brand.php

```diff
@@ -2,19 +2,42 @@
 
 namespace Lipscore\RatingsReviews\Model\System\Config\Source\Product;
 
-use \Magento\Framework\Data\Collection;
+use Magento\Framework\Data\Collection;
+use Magento\Catalog\Model\Product;
+use Magento\Eav\Model\ResourceModel\Entity\Attribute\CollectionFactory as AttributeCollectionFactory;
+use Magento\Eav\Model\Entity\Type;
+use Lipscore\RatingsReviews\Model\Logger;
```

Comment:
yes, this is exactly what I asked.
the only things is:

Do you think we still need to have these lines? 
```
Magento\Eav\Model\ResourceModel\Entity\Attribute\CollectionFactory as AttributeCollectionFactory;
use Magento\Eav\Model\Entity\Type;
use Lipscore\RatingsReviews\Model\Logger;
```

Reply:
They are used in property declaration, but I can replace them also when you want @ivanovaolga

Reply:
@ivanovaolga updated. I removed those usage and replaced the property deceleration with full path